src/api/Main/DataLayer/Shrooms.DataLayer/Migrations/201909201038299_AddIsActiveColumnToKudoType.cs

```diff
@@ -2,14 +2,14 @@ namespace Shrooms.DataLayer.Migrations
 {
     using System;
     using System.Data.Entity.Migrations;
-    
+
     public partial class AddIsActiveColumnToKudoType : DbMigration
     {
         public override void Up()
         {
-            AddColumn("dbo.KudosTypes", "IsActive", c => c.Boolean(nullable: false));
+            AddColumn("dbo.KudosTypes", "IsActive", c => c.Boolean(nullable: false, defaultValue: true));
```

Comment:
If I remember correctly, it's not enough to make it work correctly with old entries.
Can you try and confirm that migration could be successfully executed on database with existing KudosTypes?

There was another workaround implemented previously for default values (like here - https://github.com/VismaLietuva/simoona/blob/master/src/api/Main/DataLayer/Shrooms.DataLayer/Migrations/201907201330177_AddKudosWelcomeColumns.cs#L21), but I don't remember the whole background.

Reply:
My bad, I thought replying to email will get the correct conversation in thread - sadly not.

I stand corrected - seems like EF has made some magic changes and now `defaultValue` [works as expected even for old data](https://stackoverflow.com/a/51989337). I would agree that it needs confirmation, but seems like it would be fine now.

Reply:
From your Stackoverflow thread:
> "I’m not sure which version of EF is required, but it should at least work on modern ones (≥6.2)"

So yes, it should be checked and confirmed. ;)
As much as I remember in Simoona there is currently lower EF version, but probably could be upgraded if it's proven there are new useful features.

Reply:
It does work fine, I updated database to the previous migration (removed IsActive column) and ran and the after applying the last migration IsActive column was created with values set to true:
![image](https://user-images.githubusercontent.com/37371210/65977335-71543d80-e47a-11e9-9154-eef444bcf652.png)

Reply:
Sorry, but I tried to deploy to staging environment and migration failed during deployment (see below attached). Seems failure is not related to default fields, but something is still wrong with migration. Try adding new migration and check why it sees pending changes.

Please focus on this issue and fix it as soon as possible, since it's already in master.


![image](https://user-images.githubusercontent.com/2393205/65988792-2db8fe00-e491-11e9-94dd-1a45bb3f9dc8.png)

Error stack trace:
```
2019-10-01T18:16:58.4198088Z Applying explicit migrations: [201909201038299_AddIsActiveColumnToKudoType].
2019-10-01T18:16:58.4214847Z Applying explicit migration: 201909201038299_AddIsActiveColumnToKudoType.
2019-10-01T18:16:59.7890222Z VERBOSE: ALTER TABLE [dbo].[KudosTypes] ADD [IsActive] [bit] NOT NULL DEFAULT 1
2019-10-01T18:16:59.8943579Z VERBOSE: INSERT [dbo].[__MigrationHistory]([MigrationId], [ContextKey], [Model], [ProductVersion])
2019-10-01T18:16:59.8944950Z VALUES (N'201909201038299_AddIsActiveColumnToKudoType', N'Shrooms.DataLayer.ShroomsDbContext',  0x1F8B0800000000000400EDBDEB721C39B226F87FCDF61D68FC3533DB2396545DDD7DCAA419A328A98BDDA4C456AABAF69C3F69C14C2819CDC888AC8848493C6BF364F3631F695F6101C40D17C72D02714B8595992A1970381C8E0FEE8EFBFFF7BFFFDF97FFF3DB3E3AFB82D22C4CE257E7CF9FFD707E86E24DB20DE3DDABF363FEF9BFFFE5FC7FFE8FFFF3FF78F976BBFF76F6CF8AEE47428773C6D9ABF3873C3FFC7C71916D1ED03EC89EEDC34D9A64C9E7FCD926D95F04DBE4E2C50F3FFCDBC5F3E71708B338C7BCCECE5E7E3CC679B847F40FFCE755126FD0213F06D16DB24551567EC7292BCAF5EC7DB047D921D8A057E7AB873449F6D9B337411EDC044F287DF6E6F2E6FCEC320A032CCC0A459FCFCF82384EF220C7A2FEFC6B8656799AC4BBD5017F08A24F4F0784E93E075186CA2AFCDC90DBD6E68717A436174DC68AD5E698E5C9DE91E1F31F4BF55C88D95B29F9BC561F55EDFE10A16FA4DA548BAFCE6FC247945D25518436451162A13F5F4529A16F74FD16B756FE54B4CDB3EA7FC7280FBF0651F44C60F8873328DB1F6AF0608C91FFFE707685391C53F42A46C7...
2019-10-01T18:17:02.5025094Z System.Data.Entity.Migrations.Infrastructure.AutomaticMigrationsDisabledException: Unable to update database to match the current model because there are pending changes and automatic migration is disabled. Either write the pending model changes to a code-based migration or enable automatic migration. Set DbMigrationsConfiguration.AutomaticMigrationsEnabled to true to enable automatic migration.
2019-10-01T18:17:02.5025901Z    at System.Data.Entity.Migrations.DbMigrator.Upgrade(IEnumerable`1 pendingMigrations, String targetMigrationId, String lastMigrationId)
2019-10-01T18:17:02.5026135Z    at System.Data.Entity.Migrations.Infrastructure.MigratorLoggingDecorator.Upgrade(IEnumerable`1 pendingMigrations, String targetMigrationId, String lastMigrationId)
2019-10-01T18:17:02.5026345Z    at System.Data.Entity.Migrations.DbMigrator.UpdateInternal(String targetMigration)
2019-10-01T18:17:02.5026537Z    at System.Data.Entity.Migrations.DbMigrator.EnsureDatabaseExists(Action mustSucceedToKeepDatabase)
2019-10-01T18:17:02.5027569Z    at System.Data.Entity.Migrations.DbMigrator.Update(String targetMigration)
2019-10-01T18:17:02.5027898Z    at System.AppDomain.DoCallBack(CrossAppDomainDelegate callBackDelegate)
2019-10-01T18:17:02.5027998Z    at System.Data.Entity.Migrations.Console.Program.Run()
2019-10-01T18:17:02.5029450Z    at System.Data.Entity.Migrations.Console.Program.Main(String[] args)
2019-10-01T18:17:02.5029613Z ERROR: Unable to update database to match the current model because there are pending changes and automatic migration is disabled. Either write the pending model changes to a code-based migration or enable automatic migration. Set DbMigrationsConfiguration.AutomaticMigrationsEnabled to true to enable automatic migration.
```

Reply:
I opened new PR for fixing this problem.